src/SensioLabs/Connect/Api/Entity/AbstractEntity.php

```diff
@@ -137,6 +137,10 @@ public function __call($name, $arguments)
         $method = substr($name, 0, 3);
         $property = lcfirst(substr($name, 3));
 
+        if (in_array($property, ['club', 'clubsUrl', 'project', 'projectsUrl'])) {
```

Comment:
does this really work? I think not, because the C/P is going to be uppercase
we should strtolower the property

Reply:
$property is already lcfirst